2_git/week1/countries.txt

```diff
@@ -0,0 +1,3 @@
+Venice
```

Comment:
Venice is a city, the country would be Italy. But good work creating the files :)